### PR TITLE
New take on a docker setup

### PR DESCRIPTION
**Pull request by danquah**
TODOs
- Update Readme
- General feedback on structure / naming / etc
- Local testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+8. Profit! 💰
+
+Tips & tricks
+-------------
+* Output from the node container: `docker-compose logs -f node`
+* If gulp stops because of an error, you can run: `docker-compose exec node npm run gulp`
+* Changes in `package.json` should invalidate the Docker image cache.
+If it doesn't rebuild, run `docker-compose up --force-rebuild --build`
+* You can run `npm install` inside the container with `docker-compose exec node npm install`
+* Start indexing manually (starts automatically) with `docker-compose exec node run index all`
```
Comment (tripox) [inline README.md:25]: Should be removed

File: development.env-dist [hunk 2]
```diff
@@ -0,0 +1,11 @@
+GITHUB_TOKEN=my-token-from-http://
```
Comment (tripox) [inline development.env-dist:1]: Should be a real token or nothing. Maybe a comment above.

File: scripts/index-all-assets.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+docker-compose exec node npm run index all
```
Comment (tripox) [inline scripts/index-all-assets.sh:4]: This should be in the `README.md` and in a Makefile.

File: docker/dev-env-Dockerfile [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# supervisor + nginx for deployment
+RUN apt-get update && apt-get install -y \
+    libcairo2-dev \
+    libpango1.0-dev \
+    libgif-dev \
+    build-essential \
+    g++ \
+    supervisor \
+    nginx \
+&& rm -rf /var/lib/apt/lists/* # Keeps the image size down
```
Comment (tripox) [inline docker/dev-env-Dockerfile]: `apt-get clean`?

**Review by tripox — CHANGES_REQUESTED** [review]

**Comment by stumpdk** [comment]
@danquah Jeg strander når jeg kører docker-compose up. Jeg får følgende fejl: ``No such file or directory/env: bash`` fra Node-servicen

File: README.md [hunk 5]
```diff
@@ -6,16 +6,14 @@ Docker setup for kbhbilleder.
 Requirements
 ------------
 1. [Install Docker](https://store.docker.com/search?type=edition&offering=community)
-2. Clone this repository
-3. [Add an .env file to ./kbhbilleder/](https://github.com/CopenhagenCityArchives/kbh-billeder#create-a-env-file-with-environment-variables)
+2. Dory
```
Comment (tripox) [inline README.md]: Please drop a link to [the installation instructions](https://github.com/FreedomBen/dory#installation)

**Review by tripox — APPROVED** [review]

**Pull request merged**